docs/operations-guide/environment-variables.md

```diff
@@ -1,61 +1,810 @@
+
+#### `MB_PLUGINS_DIR`
+
+Type: string<br>
+Default: `"plugins"`
+
+Path of the “plugins" directory, which is used for the Metabase database drivers. The path should be to a writable directory. When using JAR, the default directory is `plugins`, created in the same location as the JAR file. When using Docker, the default directory is `/plugins`
+
+The location is where custom third-party drivers should be added. Then Metabase will load the driver on startup, which can be verified in the log.
+
+#### `MB_PREMIUM_EMBEDDING_TOKEN`
+
+Type: string<br>
+Default: `null`
+
+Token for premium features. Go to the https://store.metabase.com/ to get yours!
+
+#### `MB_QP_CACHE_BACKEND`
+
+Type: string<br>
+Default: `"db"`
+
+Current cache backend. Dynamically rebindable primary for test purposes.
+
+#### `MB_QUERY_CACHING_MAX_KB`
+
+Type: integer<br>
+Default: `1000`
+
+The maximum size of the cache, per saved question, in kilobytes.
+
+#### `MB_QUERY_CACHING_MAX_TTL`
+
+Type: integer<br>
+Default: `8640000`
+
+The absolute maximum time to keep any cached query results, in seconds. The default value is 100 days in seconds.
+
+#### `MB_QUERY_CACHING_MIN_TTL`
+
+Type: integer<br>
+Default: `60`
+
+Metabase will cache all saved questions with an average query execution time longer than this many seconds.
+
+#### `MB_QUERY_CACHING_TTL_RATIO`
+
+Type: integer<br>
+Default: `10`
+
+To determine how long each saved question's cached result should stick around, we take the query's average execution time and multiply that by whatever you input here. So if a query takes on average 2 minutes to run, and you input 10 for your multiplier, its cache entry will persist for 20 minutes.
+
+#### `MB_REPORT_TIMEZONE`
+
+Type: string<br>
+Default: `null`
+
+Connection timezone to use when executing queries. Defaults to system timezone.
+
+#### `MB_SESSION_COOKIES`
+
+Type: boolean<br>
+Default: `null`
+
+When set to `true`, the user login sessions will not expire until the browser is closed. When not set, or set to `false`, the user login sessions will expire after the amount of seconds defined in [MAX_SESSION_AGE](#MAX_SESSION_AGE) (by default 2 weeks).
+
+#### `MB_SETUP_TOKEN`
+
+Type: string<br>
+Default: `null`
+
+An UUID token used to signify that an instance has permissions to create the initial User. This is created upon the first launch of Metabase, by the first instance; once used, it is cleared out, never to be used again.
+
+#### `MB_SHOW_HOMEPAGE_DATA`
+
+Type: boolean<br>
+Default: `null`
+
+Hide the “Our data" section from the homepage by setting it to `false`. Show the section with `true`, in case it was manually removed.
+
+#### `MB_SHOW_HOMEPAGE_XRAYS`
+
+Type: boolean<br>
+Default: `null`
+
+Hide the X-rays section from the homepage by setting it to `false`. Show the section with `true`, in case it was manually removed.
```

Comment:
```suggestion
Hide the X-rays section from the homepage by setting it to `false`. Show the section with `true`, in case it was manually removed. Even if set to `true`, these will be hidden if any dashboards have been pinned in the "Our Analytics" collection.
```